requirements/default.txt

```diff
@@ -7,3 +6,5 @@ click-didyoumean>=0.3.0
 click-repl>=0.2.0
 click-plugins>=1.1.1
 importlib-metadata>=3.6; python_version < '3.8'
+backports.zoneinfo>=0.2.1; python_version < '3.9'
+tzdata>=2022.7
```

Comment:
Needed for Windows, as it doesn't have system time zone data that zoneinfo can use by default.

Details: https://tzdata.readthedocs.io/en/latest/

Reply:
also for the deprecation period you can keep this.

Reply:
Can definitely add it back - but since we've fully removed it's usage, do we possibly want it removed just to ensure it doesn't get added back?